Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -1937,6 +1991,88 @@ - (void) clearFilter
 	[listFilterField setStringValue:@""];
 }
 
+/**
+ * Refresh the current pinned tables and reload the UI
+ */
+- (void)refreshPinnedTables {
+
+    [self removePinnedTablesSection];
+    [self initPinnedTables];
+    [[tablesListView onMainThread] reloadData];
+    
+}
+
+/**
+ * Remove all currently pinned tables,
+ * Doesn't reload the UI
+ */
+- (void)removePinnedTablesSection {
+    [pinnedTables removeAllObjects];
+    NSString * pinnedHeader = NSLocalizedString(@"PINNED", @"header for pinned tables");
```

Comment:
`NSLocalizedString(@"PINNED", @"header for pinned tables");`
I believe this string is missing from the localization file!